sources/packages/backend/libs/integrations/src/sfas-integration/sims-to-sfas.integration.service.ts

```diff
@@ -0,0 +1,30 @@
+import { SFTPIntegrationBase, SshService } from "@sims/integrations/services";
+import { ConfigService } from "@sims/utilities/config";
+import { Injectable } from "@nestjs/common";
+
+import { StudentDetail } from "@sims/integrations/services/sfas";
+import { FixedFormatFileLine } from "@sims/integrations/services/ssh";
+import { SIMSToSFASFileLineBuilder } from "@sims/integrations/sfas-integration/sims-sfas-files/sims-to-sfas-file-line-builder";
```

Comment:
Should it not be in some index?

Reply:
Was this comment considered?

Reply:
Yes and I checked the other file line objects they were not having an index and were imported from the *file-line.ts from a relative path. 

Hence I changed here to a relative path import. Please let ,e know if  I am missing something here.
![image](https://github.com/user-attachments/assets/80b036e3-c283-48cf-9434-b67afefa232e)